### PR TITLE
prefer makeInvitation(..., proposalShape) to assertProposalShape

### PR DESCRIPTION
**Pull request by dckc**
follow-up to #951

drive-by: Make it easier to find ZCF APIs

### CODE REVIEW

**Comment by Chris-Hibbert** [comment]
The in-line preview makes it look broken. Can you generate the preview? This change should be fine.

**Comment by dckc** [comment]
> ... Can you generate the preview

https://dc-proposal-shape-bis.documentation-7tp.pages.dev/reference/zoe-api/zoe-helpers.html#assertproposalshape-seat-expected

I wonder why it's not as easy to find as it used to be; it's under details.

File: main/reference/zoe-api/zoe-helpers.md [hunk 1]
```diff
@@ -71,61 +75,63 @@ of the *transfer* argument of the **[atomicRearrange()](#atomicrearrange-zcf-tra
...
-    **[Amounts](/reference/ertp-api/ertp-data-types#amount)** must balance out.
- * Offer Safety is preserved for each seat. That means reallocations can only take assets from a seat
-	 as long as either it gets the assets described in the want section of its proposal, or it retains
-     all of the assets specified in the give section of the proposal. This constraint applies to each
-     seat across the entire atomicRearrangement, not to the individual **TransferParts**.
+      	Stagings are a reallocation mechanism that has been
 
-When you don't specify *toAmounts*, it means that the *fromAmount* will be taken from *fromSeat* and
-given to *toSeat*.
+  deprecated in favor of this **atomicRearrange()** function.
```
Comment (Chris-Hibbert) [inline main/reference/zoe-api/zoe-helpers.md:87]: I don't see why, but the preview site agrees with github's in-line preview that changing the indentation here broke the paragraph. 

OIC. It's [broken on docs.agoric.com](https://docs.agoric.com/reference/zoe-api/zoe-helpers.html#atomictransfer-zcf-fromseat-toseat-fromamounts-toamounts) as well.

Try indenting the whole paragraph to the same level. Are there mixed tabs and spaces here?

Comment (dckc) [reply main/reference/zoe-api/zoe-helpers.md:87]: I thought I checked that in a local build...

I must be confusing this with another PR. too many in flight...

Comment (dckc) [reply main/reference/zoe-api/zoe-helpers.md]: Ah. there were 2 occurences

**Review by Chris-Hibbert — APPROVED** [review]

**Pull request merged**